scripts/ias/create-ias-app.sh

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env bash
+
```

Comment:
in `scripts/gardener/aws/provision.sh` we have theses nice comments that explain how the script works and what it requires. Can we have something like this here too?

Reply:
added!